js-exercises/telephone-check/telephoneCheck.js

```diff
@@ -0,0 +1,13 @@
+function telephoneCheck(phoneNumber) {
+  if (typeof phoneNumber !== 'string') {
+    throw Error(`Expected String got ${typeof phoneNumber}`);
+  }
+  if (phoneNumber.length < 11) {
+    return false;
+  }
```

Comment:
okay, I have something to avoid writing 11 manually.
I want system to write that based on information of acceptable input examples.

```suggestion
 const acceptableExamples = ['555-555-5555', '(555)555-5555', '(555) 555-5555', '555 555 5555', '5555555555', '1 555 555 5555'];
 acceptableExamples.sort((a, b) => a.length - b.length);
 
  if (phoneNumber.length < acceptableExamples[0].length) {
    return false;
  }
```

Reply:
Ok. So in this question, code is expecting 1 single argument and that is the phone number. What if that phone number is invalid. We know that the length of US phone numbers is min. 11 so we should keep that in the config and check from there rather than hardcoding in the code. What do you say?

Reply:
On what information you know that 11 is minimum for US numbers.
you counted them right?
Make your system count that for you. Give system the crude information, and let it make sense from that.

Extracting count from info, and keeping it in a `config`. Yea, that's okay after above step.